app/controllers/users/passwords_controller.rb

```diff
@@ -27,26 +28,10 @@ def user_params
     end
 
     def handle_success
-      re_encrypt_active_profile
-
       bypass_sign_in current_user
 
+      flash[:recovery_code] = @update_user_password.recovery_code
```

Comment:
sorry I was a tad ambiguous: before, it was setting this flash to `active_profile.encrypt_pii(user_access_key, current_pii)` -- should we name this method something like `@update_user_password.encrypted_pii` instead?